guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -1,31 +1,21 @@
 [id="Building_a_Discovery_Image_{context}"]
-= Building a Discovery image
+= Building a custom Discovery image
 
-The Discovery image is a minimal operating system that is PXE-booted on hosts to acquire initial hardware information and to check in with {Project}.
-Discovered hosts keep running the Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process.
+You can build a custom Discovery ISO or rebuild an ISO if you change configuration files.
 
-ifdef::satellite[]
-The operating system image is based on {RHEL} 7.
-endif::[]
-ifndef::satellite[]
-Run the following procedure on {EL} 8.
+The Discovery image uses a minimal operating system that is booted on hosts to acquire initial hardware information and check in with {Project}.
+Discovered hosts keep running the Discovery operating system until they are rebooted into Anaconda, which then initiates the provisioning process.
 
-The Discovery image is based on CentOS Stream 8.
-endif::[]
+[IMPORTANT]
+====
+Do not use your production {ProjectServer} nor {SmartProxyServer} to perform this procedure.
```

Comment:
I have no idea. I've only copied it from the original doc.

Reply:
But it might be related to the system requirement that the user is not supposed to use the Foreman server for any other purposes than running Foreman. \*shrugs*